dbscript/dbscript-2.sql

```diff
@@ -3531,3 +3531,7 @@ INSERT INTO `ohrm_i18n_group` (`name`, `title`) VALUES
 ('buzz', 'Buzz'),
 ('marketplace', 'Marketplace'),
 ('mobile', 'Mobile');
+
+INSERT INTO `hs_hr_config` (`key` ,`value`) VALUES ('help.url',  'http://opensourcehelp.orangehrm.com');
```

Comment:
Can you update `http://opensourcehelp.orangehrm.com` to `https://opensourcehelp.orangehrm.com` and test